setup.py

```diff
@@ -24,6 +24,7 @@
     classifiers=[
         'Development Status :: 4 - Beta',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5'
```

Comment:
Perhaps it's a good idea to add 3.6 here because the plan is to generate packages for that version too.